scripts/task_process/cache_status.py

```diff
@@ -68,8 +73,8 @@ def insertCpu(event, info):
             info['TotalUserCpuTimeHistory'][-1] = float(event['RemoteUserCpu'])
 
 
-nodeNameRe = re.compile("DAG Node: Job(\d+(?:-\d+)?)")
-nodeName2Re = re.compile("Job(\d+(?:-\d+)?)")
+nodeNameRe = re.compile(r"DAG Node: Job(\d+(?:-\d+)?)")
+nodeName2Re = re.compile(r"Job(\d+(?:-\d+)?)")
 
```

Comment:
Are these still the same?

Reply:
I simply added `r` (means `raw`) before the string to please pylint. IIUC it tells python not to inteprete the `\` as an escape. Which it would not do in this case, but it still looks better.